### PR TITLE
feat(nlp): add autogenerated files to gitignore

### PR DESCRIPTION
**Pull request by UnbearableBear**

### CODE REVIEW

File: packages/code-du-travail-nlp/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 venv
+api/__pycache__
```
Comment (douglasduteil) [inline packages/code-du-travail-nlp/.gitignore]: Missing EOL :smirk:

**Review by douglasduteil — CHANGES_REQUESTED** [review]

**Comment by douglasduteil** [comment]
@UnbearableBear Can you confirm having an IDE that support `.editorconfig` configurations

File: packages/code-du-travail-nlp/.gitignore (hunk 1 above)
Comment (UnbearableBear) [reply packages/code-du-travail-nlp/.gitignore]: 😲

**Comment by UnbearableBear** [comment]
@douglasduteil  **** for some reasons I had not installed the vscode editorconfig extension ! (I changed my laptop recently)

**Review by douglasduteil — APPROVED** [review]

**Pull request merged**